plugin.js

```diff
@@ -5,27 +5,80 @@ var fs = require("fs");
 var path = require("path");
 var AdmZip = require("adm-zip");
 var events = require("events");
+var rmdir = require("rimraf"); // I still don't believe this is really a thing
+var pluginParser = require('./plugin-parser.js');
 
 var fileName = null;
+var localState = null;
```

Comment:
@0xPoly can you help me understand the purpose of `localState` here? I see that it is saved when delete buttons are bound (i.e. if I understand correctly after configure.js populates the plugins grid). I also see that it is used when the plugin is installed. Is this saved because otherwise you cannot access the state from `installPlugin()`? Asking because I find it quite confusing to have this local copy here.